node/rust-client/src/lib.rs

```diff
@@ -160,7 +160,7 @@ pub fn init(level: Option<Level>, file_name: Option<&str>) -> Level {
     logger_level.into()
 }
 
-fn redis_value_to_js(val: Value, js_env: Env, string_decoder: bool) -> Result<JsUnknown> {
+fn valkey_value_to_js(val: Value, js_env: Env, string_decoder: bool) -> Result<JsUnknown> {
```

Comment:
can leave it `redis_value_to_js` like the other wrappers, since it is a redis value (from redis-rs)
```suggestion
fn redis_value_to_js(val: Value, js_env: Env, string_decoder: bool) -> Result<JsUnknown> {
```